File: frontend/plant-pod-companion/src/app/plantpedia-page/plant.ts
import { NumberValueAccessor } from "@angular/forms";

export interface Plant {
    id: number;
    longName: string;
    shortName: string;
    description: string;
    minTemperature: number;
    maxTemperature: number;
    minph: number;
    maxph: number;
    care: string;
    minHumidity: number;
    maxHumidity: number;
    imageSource: string;
    moisture: string;
}

File: backend/PlantPodService/ViewModel/Plant.cs
using System;
using PlantPodService.Model;

namespace PlantPodService.ViewModel
{
    public class Plant
    {
        public Guid Id { get; set; }

        public string LongName { get; set; }
        public string ShortName { get; set; }
        public string Description { get; set; }
        public string Care { get; set; }
        public decimal MinTemperature { get; set; }
        public decimal MaxTemperature { get; set; }
        public decimal Minph { get; set; }
        public decimal Maxph { get; set; }
        public decimal MinHumidity { get; set; }
        public decimal MaxHumidity { get; set; }
        public Moisture Moisture { get; set; }
        public string Image { get; set; }
    }
}